pkgs/development/tools/knightos/scas/default.nix

```diff
@@ -12,10 +16,18 @@ stdenv.mkDerivation rec {
   };
 
   cmakeFlags = [ "-DSCAS_LIBRARY=1" ];
-
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace "TARGETS scas scdump scwrap" "TARGETS scas scdump scwrap generate_tables"
+  '';
   strictDeps = true;
 
-  nativeBuildInputs = [ cmake ];
+  depsBuildBuild = lib.optionals isCrossCompiling [ buildPackages.knightos-scas ];
+  nativeBuildInputs = [ asciidoc libxslt.bin cmake ];
```

Comment:
Do we need libxslt and asciidoc all the time or just for cross?

Reply:
It's needed for documentation